backend/migrations/20221027152757-remove-operation-status.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+ALTER TABLE operations
+  DROP COLUMN status;
+
+-- +migrate Down
+ALTER TABLE operations
```

Comment:
You probably need to add a default value here

Reply:
I didn't see one in the initial table setup or in any migrations - I guess '0' would work, since that's the previous status code number for the 'planning' stage?

Reply:
changed to '0' as the default

Reply:
Yeah, it probably didn't need a default value the first time, because no operations existed (assuming this was in the initial launch). 0 makes sense to me. We don't have room for nuance, so we're sort of stuck with limited options. 

An alternative if we didn't want to go with a default would be a 3 step approach: Add the column back as nullable, set the value for everything (again, not many options, so 0 makes sense here), then make the column non-nullable. Or variation on this.